apps/src/code-studio/pd/application_dashboard/constants.js

```diff
@@ -65,7 +65,7 @@ export const ScholarshipStatusRequiredStatuses = ['accepted'];
  * Valid statuses for this year's applications.
  * Format per application type is {value: label}
  */
-export function getApplicationStatuses(type, addAutoEmail = true) {
+export function getApplicationStatuses(type, addAutoEmail = false) {
```

Comment:
Sorry, to be clear, we want to remove the auto-email text for all components except the ones you called out in the description?

Reply:
Yes, correct! The only component that should have it (and even then, only sometimes) is DetailViewContents